srcs/layouts/latn_qwerty_cy.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard name="QWERTY (Welsh)" script="latin">
+  <row>
+    <key key0="q" key2="1" key4="loc esc"/>
+    <key key0="w" key1="~" key2="2" key3="\@" key4="ŵ"/>
+    <key key0="e" key1="!" key2="3" key3="\#" key4="ê"/>
+    <key key0="r" key2="4" key3="$"/>
+    <key key0="t" key2="5" key3="%"/>
+    <key key0="y" key2="6" key3="^" key4="ŷ"/>
+    <key key0="u" key2="7" key3="&amp;" key4="û"/>
+    <key key0="i" key2="8" key3="*" key4="î"/>
+    <key key0="o" key2="9" key3="(" key4="ô"/>
+    <key key0="p" key2="0" key3=")"/>
+  </row>
+  <row>
+    <key key0="a" key1="loc tab" key2="`"/>
+    <key key0="s" key2="loc §" key3="loc ß"/>
+    <key key0="d"/>
+    <key key0="f"/>
+    <key key0="g" key2="-" key3="_"/>
+    <key key0="h" key2="=" key3="+"/>
+    <key key0="j" key4="}" key3="{"/>
+    <key key0="k" key3="[" key4="]"/>
+    <key key0="l" key2="|" key3="\\"/>
+    <key key0="accent_circonflexe" key2="accent_trema" key3="accent_grave" />
+  </row>
+  <row>
+    <key key0="shift" key2="loc capslock"/>
+    <key key0="z"/>
+    <key key0="x" key2="loc †"/>
+    <key key0="c" key2="&lt;" key3="."/>
+    <key key0="v" key2="&gt;" key3=","/>
+    <key key0="b" key2="\?" key3="/"/>
+    <key key0="n" key2=":" key3=";"/>
+    <key key0="m" key2="&quot;" key3="'"/>
```

Comment:
`'` (key3 here) is also in the center of the next key.

Reply:
Shift and the delete key on the 3rd row are quite small. Couldn't the `'` be a swipe ?

Reply:
' is quite commonly used in Welsh, the shift and backspace are still as big as other keys.

Reply:
Alright, then `'` should be removed from the corner of the `m` key. Making Shift and Backspace the same size as the other keys is fine but as they are more frequently used, having them bigger improves typing.

Reply:
Ok, I will remove it from the `m` key.

Reply:
Removed